src/Autodoc.zig

```diff
@@ -927,14 +875,17 @@ const DocData = struct {
             }
             switch (self) {
                 .int => {
-                    if (self.int.negated) try w.writeAll("-");
-                    try jsw.emitNumber(self.int.value);
```

Comment:
I'm pretty sure this was broken, because the old `emitNumber()` would render large integers as quotes, resulting in `-"12345678901234567890"`. The new code will render it as `"-12345678901234567890"`.